### PR TITLE
Add a Slider widget (when value is 'slider', 'slider:max' or 'slider:min:max')

### PR DESCRIPTION
**Pull request by fgallaire**
With tickmarks when value ends with ':'

### CODE REVIEW

**Comment by PierreRaybaut** [comment]
Besides a quite obvious merge conflict (no need to worry about that), there would be two things to fix about this PR, to my opinion:
- when the user is moving sliders, there is no way to know what value has been set (there should be a QLineEdit, on the right, for example, to show the value interactively)
- I noticed a traceback relative to xml serialization when running `advanced.py`

**Comment by fgallaire** [comment]
@PierreRaybaut, you're right, I have chosen the simplest solution, but it's not good enough.

**Comment by fgallaire** [comment]
Don't care about XML at the moment.

File: formlayout.py [hunk 1]
```diff
@@ -265,6 +265,39 @@ def text(self):
...
+            self.slider.setTickPosition(2)
+            parsed.pop(-1)
+        if len(parsed) == 2:
+            self.slider.setMaximum(int(parsed[1]))
+        elif len(parsed) == 3:
+            self.slider.setMinimum(int(parsed[1]))
+            self.slider.setMaximum(int(parsed[2]))
+        if default:
+            self.slider.setValue(default)  # always set value in last
+        self.slider.valueChanged.connect(self.update)
```
Comment (PierreRaybaut) [inline formlayout.py]: **PyQt API 1 compatibility**: we need to handle the compatibility with old-style signals and slots. In this case, I'm expecting something like:

``` python
        if SIGNAL is None:
            self.slider.valueChanged.connect(self.update)
        else:
            self.connect(self.slider, SIGNAL("valueChanged(int)"), self.update)  # need to be tested!
```

**Comment by PierreRaybaut** [comment]
See my inline comment [here](https://github.com/PierreRaybaut/formlayout/pull/14/files)

**Pull request merged**